Portal/src/Datahub.Infrastructure/Queues/Messages/EmailRequestMessage.cs

```diff
@@ -24,6 +27,21 @@ public class EmailRequestMessage : IRequest
     public string? TemplateData { get; set; }
 
     public bool IsValid => To?.Count > 0;
+
+    [JsonIgnore]
+    public string Content 
+    { 
+        get
+        {
+            return JsonConvert.SerializeObject(this, new JsonSerializerSettings
+            {
+                ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
+            });
+        }
+    }
+
+    [JsonIgnore]
+    public string ConfigPathOrQueueName { get => "email-notification"; }
```

Comment:
I'm slightly confused, is this for backwards compatibility with the queues while we migrate fully to Mass Transit?